codalab/server/json_api_plugin.py

```diff
@@ -0,0 +1,110 @@
+"""
```

Comment:
You're right, I've decided against implementing anything more than necessary at this moment until we need it.